CONTRIBUTING.md

```diff
@@ -61,25 +61,6 @@ yarn prepare
 
 5.  **Setup environmental variables:** To run E2E Tests, you should set the following variables in your `.env` file in the repository’s root.
 
-```bash
```

Comment:
since we removed all the envvars, do we still need step 5 here?

Reply:
In that case, then we should update step 5 to remove "the following variables" (as no variables are following)... maybe make it  something like "To run E2E Tests, you should set the requisite environmental variables in your `.env` file" ?

Reply:
Updated